odd-collector/odd_collector/adapters/postgresql/tests/conftest.py

```diff
@@ -0,0 +1,258 @@
+        numeric_precision=None, numeric_precision_radix=None, numeric_scale=None, datetime_precision=None,
+        interval_type=None, interval_precision=None, character_set_catalog=None, character_set_schema=None,
+        character_set_name=None, collation_catalog=None, collation_schema=None, collation_name=None,
+        domain_catalog=None, domain_schema=None, domain_name=None, udt_catalog='test', udt_schema='public',
+        udt_name='vector', scope_catalog=None, scope_schema=None, scope_name=None, maximum_cardinality=None,
+        dtd_identifier='4', is_self_referencing='NO', is_identity='NO', identity_generation=None,
+        identity_start=None, identity_increment=None, identity_maximum=None, identity_minimum=None,
+        identity_cycle='NO', is_generated='NEVER', generation_expression=None, is_updatable='YES',
+        description=None, type_oid=16420, enums=[], is_primary=False
+    )
+
+
+@pytest.fixture(scope="module")
+def table_without_vector_column(int4_id_column, varchar_column):
+    return Table(
+        oid=16520,
+        table_catalog='test',
+        table_schema='test',
+        table_name='test',
+        table_type='r',
+        self_referencing_column_name=None,
+        reference_generation=None,
+        user_defined_type_catalog=None,
+        user_defined_type_schema=None,
+        user_defined_type_name=None,
+        is_insertable_into='YES',
+        is_typed='NO',
+        commit_action=None,
+        view_definition=None,
+        view_check_option=None,
+        view_is_updatable=None,
+        view_is_insertable_into=None,
+        view_is_trigger_updatable=None,
+        view_is_trigger_deletable=None,
+        view_is_trigger_insertable_into=None,
+        table_owner='postgres',
+        table_rows=100,
+        description=None,
+        columns=[int4_id_column, varchar_column, ],
+        primary_keys=[]
+    )
+
+
+@pytest.fixture(scope="module")
+def table_with_vector_column(int4_id_column, varchar_column, vector_column):
+    return Table(
+        oid=16520,
+        table_catalog='test',
+        table_schema='test',
+        table_name='test',
+        table_type='r',
+        self_referencing_column_name=None,
+        reference_generation=None,
+        user_defined_type_catalog=None,
+        user_defined_type_schema=None,
+        user_defined_type_name=None,
+        is_insertable_into='YES',
+        is_typed='NO',
+        commit_action=None,
+        view_definition=None,
+        view_check_option=None,
+        view_is_updatable=None,
+        view_is_insertable_into=None,
+        view_is_trigger_updatable=None,
+        view_is_trigger_deletable=None,
+        view_is_trigger_insertable_into=None,
+        table_owner='postgres',
+        table_rows=100,
+        description=None,
+        columns=[int4_id_column, varchar_column, vector_column, ],
+        primary_keys=[]
+    )
```

Comment:
Your `table_with_vector_column` basically is the same table from another fixture but with extra column. I believe you don't have copypaste the whole construct and just something like this:
```suggestion
def table_with_vector_column(table, vector_column):
    # pseudo code
    table.columns << vector_column
    return table
```

Reply:
@AndreyNenashev done.